buildSrc/DevBuild.js

```diff
@@ -17,6 +17,7 @@ export async function runDevBuild({ stage, host, desktop, clean, ignoreMigration
 	if (clean) {
 		await runStep("Clean", async () => {
 			await fs.emptyDir("build")
+			await fs.emptyDir("build-calendar-app")
```

Comment:
I remember we talked about it before but was there a reason why there's no separate make file for calendar?

Reply:
No specific reason